### PR TITLE
feat: about page gets meaningful content

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
[MUMUP-3312](https://jira.doit.wisc.edu/jira/browse/MUMUP-3312): "As a user following the About link in the sidebar menu, I'd like useful information answering whatever question I had that made me go to 'About', so that my questions are answered."

**In this PR:**
- uportal-app-framework version info and app-specific info moved to "Session info" page (linked to from "About" page)
- "About" page populated with text/links from a JSON source
- Updated docs to explain configuration of new feature
- Removed unused "demo widgets" page and path
- Removed frustrating, baffling, nonsensical remark-lint "list-item-indent" rule so that I can submit merge requests without unfixable false positives ruining the build

### Screenshots
screen shot 2018-05-04 at 11 15 08 am
screen shot 2018-05-04 at 11 42 20 am

----

PR considerations checklist:

- [x] Updates `CHANGELOG.md` to reflect this PR's change.
- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: components/staticFeeds/about-frame.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
```
Comment (davidmsibley) [inline components/staticFeeds/about-frame.json]: This has a different structure than the filtered-resources/about-frame.json, may need updating?

File: components/portal/misc/partials/example-menu.html [hunk 2]
```diff
@@ -58,10 +58,18 @@
     </md-button>
   </md-menu-item>
 <md-menu-item>
-  <md-button href="/about" ng-click="vm.closeMainMenu();" layout="row" layout-align="start center">
+  <md-button href="about" ng-click="vm.closeMainMenu();" layout="row" layout-align="start center">
```
Comment (davidmsibley) [inline components/portal/misc/partials/example-menu.html]: did you mean to make this href relative if session-info is off the root?

File: components/staticFeeds/about-frame.json (hunk 1 above)
Comment (thevoiceofzeke) [reply components/staticFeeds/about-frame.json]: Reverted the change to filtered-resources/about-frame.json so they are now identical

File: package.json [hunk 3]
```diff
@@ -190,7 +190,7 @@
       "validate-links",
       [
         "remark-lint-list-item-indent",
-        "mixed"
```
[the remark this answers is not in the document]
Comment (thevoiceofzeke) [reply package.json]: It stops enforcing the inane and often infuriating list-item-indent markdown rule that causes incomprehensible errors every time I run remark-lint. Probably should have checks with folks first, but I really don't think we need that rule.

**Pull request merged**